mkt/webapps/cron.py

```diff
@@ -564,22 +569,26 @@ def mkt_gc(**kw):
     Nonce.objects.filter(created__lt=days_ago(1)).delete()
 
     # Delete the dump apps over 30 days.
-    _remove_stale_files(settings.DUMPED_APPS_PATH,
+    _remove_stale_files(os.path.join(settings.DUMPED_APPS_PATH, 'tarballs'),
```

Comment:
Note for posterity: it was actually broken before, and we could retrieve all older dumps, because `tarballs` was not included, so `_remove_stale_files()` was listing the parent dir, only finding the `tarballs` directory, which always had a recent modification date since there are new dumps put in there every day.